lib/session/session.go

```diff
@@ -276,20 +286,43 @@ func activeKey(namespace string, key string) []byte {
 
 // GetSessions returns a list of active sessions. Returns an empty slice
 // if no sessions are active
-func (s *server) GetSessions(namespace string) ([]Session, error) {
-	prefix := activePrefix(namespace)
+func (s *server) GetSessions(namespace string, cond *types.WhereExpr) ([]Session, error) {
```

Comment:
I would refrain from introducing nullable parameters in the API method signature as it makes a method call no longer self-descriptive. Since we are changing input parameters anyway, it might be a good time to introduce  `GetSessionRequest`  (also more scallable)

Reply:
Agreed - I didn't want to change too much of the original commits, but this struck out to me as well. Having `cond` as a param seems to make it easy to (inadvertently) skip permission checks too.

Jane had similar comments as well - I'll do a few changes and let you know when ready.

Reply:
Done.